src/hb-aat-map.cc

```diff
@@ -66,10 +67,18 @@ void hb_aat_map_builder_t::add_feature (hb_tag_t tag, unsigned value)
     else return;
   }
 
+  unsigned int default_index;
+  hb_aat_layout_feature_type_get_selector_infos (face,
```

Comment:
No need to call public API.  That API simply calls `face->table.feat->get_selector_infos`.  Do instead.  Also, maybe add `is_exclusive` to feat table.

Reply:
Done.